recipes/itango/LICENSE

```diff
@@ -0,0 +1,166 @@
+
```

Comment:
This file can be removed as it's included in the source tarball (no need to change anything else)

Reply:
The LICENSE was submitted upstream and [merged](https://github.com/tango-controls/itango/pull/22) but no new version has been released yet. The LICENSE isn't part of version 0.1.7.